pkg/operator/operator.go

```diff
@@ -253,42 +250,28 @@ func (o *operator) Run(ctx context.Context) error {
 			if rErr != nil {
 				return rErr
 			}
-			healthzServer.Ready()
+			o.tshealthz[0].Ready()
```

Comment:
Ideally we can have a more explicit reference to the server instead of using an index

Reply:
I agree- I'll make as named vars.

Reply:
Done!